### ISSUE TITLE
ConditionFileExists=/var/lib/obmc/cfam_overrides not valid in cfam_override@.service

### ISSUE THREAD

**Issue by geissonator** [openbmc/openbmc#1582]
Don't think there's any such thing as ConditionFileExists.

Probably want ConditionFileNotEmpty=
or
ConditionPathExists=

Saw this error when doing some other testing:
```
May 01 18:39:20 witherspoon systemd[1]: [/lib/systemd/system/obmc-chassis-poweron@0.target.requires/../cfam_override@.service:11] Unknown lvalue 'ConditionFileExists' in section 'Unit'
```

**Comment by mtritz** [comment]
- [ ] Change ConditionFileExists to ConditionFileNotEmpty

**Comment by rfrandse** [comment]
https://gerrit.openbmc-project.xyz/4164 cfam_override@.service - run on ConditionPathExists
Resolves #1582 ConditionFileExists=/var/lib/obmc/cfam_overrides not valid in cfam_override@.service
